vtex-sites/base.store
# chore: update faststore version

## What's the purpose of this pull request?
This PR:
- Updates faststore to version 1.4.5
- Fix a callback placed in the wrong scope at analytics tests
- Updates data attribute names on stylesheets to sync with https://github.com/vtex/faststore/pull/1093

## How to test it?
`yarn test`. Verify that all tests pass.
`yarn develop`. Verify that the styles behave like intended and nothing is "visually broken"

## References
@hellofanny 
@renatamottam 
Thanks for everything!
